ociclient/test/envtest/envtest.go

```diff
@@ -117,14 +120,19 @@ func (e *Environment) Start(ctx context.Context) error {
 }
 
 func (e *Environment) Close() error {
+	e.mu.RLock()
 	if !e.stopped {
 		e.cancelCtx()
 	}
+	e.mu.RUnlock()
 	// wait until the process is stopped.
 	for {
+		e.mu.RLock()
```

Comment:
```suggestion
		e.mu.RLock()
		stopped := e.stopped
		e.mu.RUnlock()
		if stopped {
```

Reply:
Yeah, I thought about that too, but wasn't sure. I changed it.